src/BugsnagUnity/Payload/StackTraceLine.cs

```diff
@@ -45,6 +45,10 @@ internal StackTrace(string stackTrace, StackTraceFormat format)
                   : StackTraceLine.FromLogMessage(lines[i]);
                 if (frame != null)
                 {
+                    if (frame.Method.StartsWith("at "))
+                    {
+                        frame.Method = frame.Method.TrimStart('a', 't', ' ');
+                    }
```

Comment:
I'm not really sure what case this is handling, maybe one for the [unit tests](https://github.com/bugsnag/bugsnag-unity/blob/next/tests/BugsnagUnity.Tests/StackFrameParsingTests.cs)?

Reply:
Android stacktraces coming through the new handler had "at " at the beginning of each frame method. It's coverd in CI tests. Is this ok?